src/main/java/build/buildfarm/common/CasIndexResults.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buildfarm.common;
+
+///
+/// @class   CasIndexResults
+/// @brief   The result of a worker de-registering itself from CAS entries.
+/// @details This will remove the worker from CAS indexes, and in some cases,
+///          remove the CAS key.
+///
+public class CasIndexResults {
+
+  ///
+  /// @field   removedInstances
+  /// @brief   The number of CAS entries the worker was removed from.
+  /// @details This indicates how much CAS data the shard new the worker had.
+  ///
+  public int removedInstances = 0;
```

Comment:
Let's call them hosts here. Instance is probably too AWS specific.

Reply:
fix